link-grammar-server.bat

```diff
@@ -10,6 +10,9 @@ java ^
 %DISCO_HOME%\lib\gnu-getopt.jar;^
 %DISCO_HOME%\lib\wordnet\jwnl.jar;^
 %DISCO_HOME%\lib\wordnet\commons-logging.jar;^
+%DISCO_HOME%\lib\wordnet\slf4j-api-1.7.25.jar;^
```

Comment:
Don't think logging related jars should be placed in ```wordnet``` folder.

Reply:
It is the question what is the right place to put logging jar on Windows?
I see that commons-logging.jar is placed in the wordnet directory.